### ISSUE TITLE
Some visual bugs in the 20210714 update

### ISSUE THREAD

**Issue by LeFloatyPancake** [daniruiz/skeuos-gtk#18]
This was tested on Nautilus 40.2 and Skeuos-grey-light

I have been comparing the latest release (20210714) with the previous one (20210706) and I found some visual glitches. Please see the image below. 

Edit: I did further testing on other GTK apps such as font-manager and gnome-system-monitor. There seems to be an issue with highlight background of selected item and the contents inside it being almost unreadable in the 20210714 update.

![bug01](https://user-images.githubusercontent.com/83509112/125642975-bf3c3ceb-c137-45c8-b46a-81b7534e2b33.png)

**Comment by daniruiz** [comment]
Hi!
Yes I notices it just after pushing the new release :sweat_smile: 
I'm already working on it
Thanks for the report btw :heart:

**Comment by daniruiz** [comment]
Hello again!
I've pushed a new update to github which should fix these issues.
I've made the selected color a bit more contrasted but I do want to keep this new design as for me it looks way more modern than using a solid color.
If you can give me extra feedback that would help me a lot :wink: 
Thank you!

**Comment by LeFloatyPancake** [comment]
Thanks for the notif. I expanded the testing of the latest updates to blue, yellow, and cyan variants for light themes. The softer selection bg does seem lighter on the eyes as I was testing it on geary and evolution with some variants such as cyan and grey having a selected bg that blended to close to the background behind it (might just be my wonky vision). 

Anyway I found other things that could use some readability tweaks as can be seen on the screenshots below. The theme used here was the latest `skeuos-blue-light`. GTK Widget Factory did not really give me much to differentiate so I used nautilus, gnome system monitor, and font-manager. 

The UI fixes for nautilus worked like a charm. Thanks again.
 
![bug02](https://user-images.githubusercontent.com/83509112/125942400-58edad8e-7635-4173-afb7-46c1f38323fd.png)
